session.go

```diff
@@ -344,7 +354,7 @@ func (session *Session) startCheckBytesRead() error {
 		session.RLock()
 		sentTime := session.bytesReadSentTime
 		updateTime := session.bytesReadUpdateTime
-		bytesRead := session.bytesRead
+		bytesRead := session.bytesReceived // return byte received by session
```

Comment:
Why? Data should be removed from sender buffer when receiver read it, not receive it

Reply:
Restored